prediction_market_agent/agents/microchain_agent/microchain_agent.py

```diff
@@ -156,6 +157,7 @@ def build_agent(
             api_key=keys.openai_api_key.get_secret_value(),
             api_base=api_base,
             temperature=0.7,
+            enable_langfuse=enable_langfuse,
```

Comment:
put a comment on the microchain pr with an idea on how to remove the need to specify this flag here https://github.com/galatolofederico/microchain/pull/18/files#r1726981146

Reply:
It was a good idea, but I forgot that enable_langfuse property is required also because of this:

![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/700434ec-3c36-4932-a9b8-29c3a52778e6)

So I just opened a PR fixing it 😄